.gitignore

```diff
@@ -26,7 +26,7 @@ npm-debug.log
 # IDE Directories #
 .idea
 !.idea/codeStyleSettings.xml
-!.idea/ngx-material-keyboard.iml
+!.idea/angular-onscreen-material-keyboard.iml
```

Comment:
Daniel, I cannot accept the pull request because you changed the package name all over the place. It  would be fine if you just submit a pull request only containing  src/core/src/components/keyboard-key/keyboard-key.component.ts